src/Microsoft.ApplicationInsights.AspNetCore/Logging/Implementation/ApplicationInsightsLoggerOptions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.ApplicationInsights.AspNetCore.Logging
+{
+    /// <summary>
+    /// Options for configuration of <see cref="ApplicationInsightsLogger"/>.
+    /// </summary>
+    public class ApplicationInsightsLoggerOptions
```

Comment:
There is another PR which introduces ```ApplicationInsightsLoggerOptions```
https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/574/files
Can you also have took at that as well?

I think passing ```ApplicationInsightsLoggerOptions``` to both ApplicationInsightsLoggerProvider and ApplicationInsightsLogger as done in the other PR is a better option. Can you incorporate this change to your PR?

Reply:
Thank you for the hint. Yes, it's actually better. Also I realized that classes are internal, so I can change signature of constructors.